clang-tools-extra/docs/clang-tidy/ExternalClang-TidyExamples.rst

```diff
@@ -5,21 +5,22 @@ External Clang-Tidy Examples
 Introduction
 ============
 
-This page provides examples of what people have done with clang-tidy that 
+This page provides examples of what people have done with `clang-tidy`` that 
 might serve as useful guides (or starting points) to develop your own checks. 
-They may be helpful even for necessary things such as how to write CMakeLists.txt 
-for an out-of-tree plugin of clang-tidy checks.
+They may be helpful for necessary things such as how to write CMakeLists.txt 
+for an out-of-tree plugin of `clang-tidy` checks.
 
-If you know of (or wrote!) a tool or project using clang-tidy, please post on
-`the Discourse forums (Clang Frontend category)
-<https://discourse.llvm.org/c/clang/6>`_ to have it added.
-(or if you are already a clang-tidy contributor, feel free to directly commit
-additions). Since the primary purpose of this page is to provide examples
-that can help developers, generally they must have code available.
+If you know of (or wrote!) a tool or project using clang-tidy, please share it 
+on `the Discourse forums (Clang Frontend category)
+<https://discourse.llvm.org/c/clang/6>`_ for wider visibility and open a 
+pull-request on `LLVM Github`_ to have it added here. Since the primary purpose of 
+this page is to provide examples that can help developers, generally they must have 
```

Comment:
Maybe instead of `..., generally they must have...` 
->

`..., the listed projects should be open-source.`
or
`..., they generally must have code available`
or
`..., they should have code available.`